src/form/GeometryField.tsx

```diff
@@ -82,59 +42,66 @@ export const GeometryField: React.FC<MyProps> = (props) => {
       <span className={formStyles.LabelTitle}>
         {title}
       </span>
-      <div style={{display: "flex", }}>
-        
+      <div
+        style={{
+          display: "flex",
+          color: "#C9C9C9"
+        }}
+      >
         {/* lng */}
         <div style={{marginRight: "16px", }}>
           <FloatInput
-            title={"x"}
+            title={"X"}
             name={name}
             placeholder={placeholder}
-            value={lng}
-            validated={validated}
-            valueChanged={(value)=>{
-              if (isNaN(value)) {
-                setLng(NaN);
-              } else {
-                valueChangedLng(value);
-              }
-              
+            value={value.location ? value.location.lng : NaN}
+            valueChanged={e => {
+              if (isNaN(e)) return;
```

Comment:
Not super important, but now when the user empties the field it becomes 0. Previously the field was allowed to be really empty.

Reply:
Yes, I will fix this